doc/Programming-guide/Your-first-Kaa-application/index.md

```diff
@@ -384,13 +384,130 @@ int main(void)
 
     return EXIT_SUCCESS;
 }
-
 ```
 
 </div>
-
-<div id="app-c++" class="tab-pane fade" markdown="1" >
-
+ 
+<div id="C_plus_plus" class="tab-pane fade" markdown="1" >
```

Comment:
I don't see changes in "Preparation" and "Launching application" sections. Please attach.